cmd/crc/cmd/daemon_windows.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"net"
+
+	"github.com/Microsoft/go-winio"
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+)
+
+func controlListener() (net.Listener, error) {
```

Comment:
need a `httpListener()` wrapper over this, build fails otherwise

```diff
diff --git a/cmd/crc/cmd/daemon_windows.go b/cmd/crc/cmd/daemon_windows.go
index a188d301..bbdfd313 100644
--- a/cmd/crc/cmd/daemon_windows.go
+++ b/cmd/crc/cmd/daemon_windows.go
@@ -20,3 +20,7 @@ func controlListener() (net.Listener, error) {
        }
        return ln, nil
 }
+
+func httpListener() (net.Listener, error) {
+       return controlListener()
+}
```

Reply:
thanks! fixed